drivers/gles3/rasterizer_storage_gles3.cpp

```diff
@@ -7115,7 +7115,7 @@ void RasterizerStorageGLES3::_render_target_allocate(RenderTarget *rt) {
 	bool hdr = rt->flags[RENDER_TARGET_HDR] && config.framebuffer_half_float_supported;
 	//hdr = false;
 
-	if (!hdr || rt->flags[RENDER_TARGET_NO_3D]) {
+	if (!hdr && rt->flags[RENDER_TARGET_NO_3D]) {
```

Comment:
This logic is slightly incorrect. 

Right now, this forces HDR for every texture using ``3D`` or ``3D no effects``.